src/Form/FormControl.js

```diff
@@ -9,7 +9,7 @@ import { isDirty } from '../Input/Input';
 
 export const styleSheet = createStyleSheet('MuiFormControl', theme => ({
   root: {
-    display: 'flex',
+    display: 'inline-flex',
```

Comment:
Breaking change

Reply:
@oliviertassinari Would be good to see this breaking change in changelog. All our inputs are broken now. We didn't expect the fire from minor upgrade.